projects/schematics/src/add-spartacus/index_spec.ts

```diff
@@ -20,41 +20,41 @@ import { Schema as SpartacusOptions } from './schema';
 
 const collectionPath = path.join(__dirname, '../collection.json');
 
-describe('add-spartacus', () => {
-  const schematicRunner = new SchematicTestRunner(
-    SPARTACUS_SCHEMATICS,
-    collectionPath
-  );
-
-  let appTree: UnitTestTree;
-
-  const workspaceOptions: WorkspaceOptions = {
-    name: 'workspace',
-    newProjectRoot: 'projects',
-    version: '0.5.0',
-  };
-
-  const appOptions: ApplicationOptions = {
-    name: 'schematics-test',
-    inlineStyle: false,
-    inlineTemplate: false,
-    routing: false,
-    style: Style.Scss,
-    skipTests: false,
-    standalone: false,
-  };
-
-  const defaultOptions: SpartacusOptions = {
-    project: 'schematics-test',
-    occPrefix: 'xxx',
-    baseSite: 'electronics',
-    baseUrl: 'https://localhost:9002',
-    lazy: true,
-    features: [],
-  };
-
-  const newLineRegEx = /(?:\\[rn]|[\r\n]+)+/g;
+const schematicRunner = new SchematicTestRunner(
+  SPARTACUS_SCHEMATICS,
+  collectionPath
+);
+
+let appTree: UnitTestTree;
+
+const workspaceOptions: WorkspaceOptions = {
+  name: 'workspace',
+  newProjectRoot: 'projects',
+  version: '0.5.0',
+};
+
+const appOptions: ApplicationOptions = {
+  name: 'schematics-test',
+  inlineStyle: false,
+  inlineTemplate: false,
+  // routing: true,
```

Comment:
`nitpick:` Is this line commented on intentionally? or should we remove this part of the options?